kvs/client/python/anna/ipc_client.py

```diff
@@ -12,59 +12,167 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-GET_ADDR = "ipc:///requests/get"
-PUT_ADDR = "ipc:///requests/put"
+GET_REQUEST_ADDR = "ipc:///requests/get"
+PUT_REQUEST_ADDR = "ipc:///requests/put"
 
+GET_RESPONSE_ADDR_TEMPLATE = "ipc:///responses_%d/get"
+PUT_RESPONSE_ADDR_TEMPLATE = "ipc:///responses_%d/put"
+
+TIMEOUT = 5000
+
+from .functions_pb2 import *
 from .kvs_pb2 import *
 from .lattices import *
 import zmq
 
 class IpcAnnaClient:
-    def __init__(self, offset=0):
+    def __init__(self, thread_id = 0, offset=0):
```

Comment:
The `offset` is a `thread_id`. :-) But `thread_id` is a better name -- let's remove `offset`.

Reply:
fixed